src/runtime/lpython_builtin.py

```diff
@@ -695,6 +695,19 @@ def _lpython_str_lower(x: str) -> str:
             res += i
     return res
 
+@overload
+def _lpython_str_upper(x: str) -> str:
+    res: str
+    res = ""
+    i:str
+    for i in x:
+        if ord('a') <= ord(i) and ord('z') >= ord(i):
```

Comment:
**Minor:**

I usually write this type of conditions as:
```python
 if ord('a') <= ord(i) and ord(i) <= ord('z') :
```
So, in a quick glance the condition looks like `ord('a') <= ord(i) <= ord('z')` and thus is (more) clearer to read/interpret.

(In case you update the above condition, please also update for `_lpython_str_lower()`.)

Reply:
Thanks for the suggesting. Ill fix this :)